app/server.ts

```diff
@@ -124,10 +124,11 @@ app.use(
 
 async function initializeDatabase(): Promise<void> {
   const db = getInstance(config.DB_PATH);
+
   const numberOfTables = (await db.listTables()).length;
   if (numberOfTables === 0) {
     await db.applyMigrationScripts(
-      path.join(__dirname, 'migrations', 'schema')
+      path.join(__dirname, 'migrations', 'schema_pg')
```

Comment:
Should select schema based on `DB_PATH`

Reply:
true - going to edit this, it was the first implementation eheh